templatestore/views.py

```diff
@@ -95,6 +97,7 @@ def get_templates_view(request):
                     "type": t.type,
                     "attributes": t.attributes,
                     "created_on": t.created_on,
+                    "modified_on": t.modified_on,
```

Comment:
update this also: https://github.com/wilspi/django-templatestore/blob/master/docs/swagger.json